shell/browser/ui/cocoa/atom_bundle_mover.mm

```diff
@@ -70,11 +90,22 @@
       }
     }
   } else {
+    base::OnceCallback<bool(ConflictType)> conflict_cb;
```

Comment:
What happens if this method returns something that isn't a `bool` (like a string or something).  I have a terrible feeling it's gonna throw a weird mate error or something

Reply:
yeah, i think this is gonna need to be 
```
base::OnceCallback<v8::MaybeLocal<v8::Boolean>(ConflictType)> conflict_cb;
```